extensions/amp-viewer-gpay-button/amp-viewer-gpay-button.md

```diff
@@ -1,28 +1,11 @@
-<!--
-  1. Change "category" below to one of:
-       ads-analytics
-       dynamic-content
-       layout
-       media
-       presentation
-       social
-
-  2. Remove any of the "formats" that don't apply.
-     You can also add the "ads" and "stories" formats if they apply.
-
-  3. And remove this comment! (no empty lines before "---")
--->
-
 ---
-
-\$category: presentation
```

Comment:
Nice catch. I noticed `gulp prettify` allows `$category` only if it's at the top of the file and corrects it to `\$category` if it isn't. I assume this is what you're referring to as frontmatter?